packages/calypso-products/src/plans-list.tsx

```diff
@@ -847,6 +850,10 @@ const getPlanPersonalDetails = (): IncompleteWPcomPlan => ( {
 			features = [ FEATURE_UPLOAD_PLUGINS_SUMMER_SPECIAL, ...features ];
 		}
 
+		if ( isGlobalStylesGridChangesVariation() ) {
+			features = [ ...features, FEATURE_WORDADS, FEATURE_PAYMENT_TRANSACTION_FEES_8 ];
```

Comment:
One last thing I found, we shouldn't add `FEATURE_WORDADS` to the Personal plan, since it requires the Premium plan.